projects/nae-example-app/src/index.html

```diff
@@ -6,6 +6,7 @@
     <base href="/">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="icon" type="image/x-icon" href="favicon.ico">
+    <script src="./assets/env.js"></script>
```

Comment:
For importing scripts, use a relative URL `assets/env.js`, not a file system relative path.